users/config.rst

```diff
@@ -899,6 +899,22 @@ The following child elements may be present:
     ``ldap``
         LDAP authentication. Requires ldap top level config section to be present.
 
+.. option:: gui.sendBasicAuthPrompt
+
+    The GUI previously used HTTP Basic Authorization for login, but now uses an
```

Comment:
Just a quick comment, as I didn't have time to look at the PR before, but "previously" can mean literally anything. How about adding a specific version number here?

Reply:
Good catch! Agreed and fixed.